enterprise/replay/fileuploader/s3downloader.go

```diff
@@ -15,10 +15,21 @@ import (
 func Download(file *os.File, key, bucket string) (int64, error) {
 	getRegionSession := session.Must(session.NewSession())
 	region, err := s3manager.GetBucketRegion(aws.BackgroundContext(), getRegionSession, bucket, "us-east-1")
-	downloadSession := session.Must(session.NewSession(&aws.Config{
-		Region:      aws.String(region),
-		Credentials: credentials.NewStaticCredentials(config.GetEnv("AWS_ACCESS_KEY_ID", ""), config.GetEnv("AWS_SECRET_ACCESS_KEY", ""), ""),
-	}))
+	if err != nil {
+		return 0, err
+	}
+
+	awsAccessKeyId := config.GetEnv("AWS_ACCESS_KEY_ID", "")
+	awsSecretAccessKey := config.GetEnv("AWS_SECRET_ACCESS_KEY", "")
```

Comment:
Wouldn't it make sense to just return an error if these are empty since logging in with an empty key or secret is simply not possible?

Reply:
Oh OK, that's good to know :+1: Btw, I meant that it's not possible to login to AWS if you don't have both an access key and a secret. Sounds right to you?